app/etl/item/content/parsers/body_content.rb

```diff
@@ -1,6 +1,11 @@
 class Item::Content::Parsers::BodyContent
   def parse(json)
-    json.dig("details", "body")
+    parsed = json.dig("details", "body")
+    if parsed.present? && parsed.is_a?(Array)
+      parsed = Hash[*parsed.map(&:values).flatten].fetch("text/html", nil)
```

Comment:
This line was quite hard for me to understand just from reading it.

Can you split it into multiple statements? I think you can also lose the splat operator if you do `parsed.map(&:values).to_h`.

Something like

```ruby
content_by_content_type = parsed.map(&:values).to_h
content = content_by_content_type.fetch("text/html", nil)
```

Reply:
Addressed in 1f365079d5efd3af103d8c8dc07eeda52ab49b63